content/theme/hugo-academia.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Academia Hugo
+github: https://github.com/themefisher/academia-hugo
+demo: https://demo.gethugothemes.com/academia/
+author: Themefisher
+ssg:
+  - Hugo
+cms:
+  - Forestry
```

Comment:
I would also add `- No Cms` to the cms array for all your themes. 

No Cms is basically "git" or "markdown" and all your themes will also work with markdown.